maquetado/scss/tech.scss

```diff
@@ -0,0 +1,64 @@
+$tech-logo-size: 100px;
+
+.technology {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 320px;
+  margin: 15px;
+  width: 230px;
+  position: relative;
+
+  &::before {
+    border: 1px solid $green;
+    border-radius: 5px;
+    bottom: 10px;
+    content: " ";
+    height: 320px;
+    position: absolute;
+    right: 10px;
+    width: 230px;
+    z-index: -1;
+  }
+}
+
+.logo,
+.technology-title,
+.technology-release,
+.technology-description {
+  margin: 10px 0;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+}
+
+.technologies {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin: 0 380px;
+}
+
+.technology-title,
+.technology-release {
+  font-size: $text-medium
```

Comment:
Move this inside
```.logo,
.technology-title,
.technology-release,
.technology-description {
```

It doesnt mind if not all the classes must have text medium, the rest will be over-written by their own styles